### PR TITLE
Em user dashboard2

### PR DESCRIPTION
**Pull request by emestabillo**
UI for user dashboard
- Design: https://www.figma.com/design/S2FpOkJ8v3PHaiVDvBhJ5E/Secret-Santa?node-id=52-14&t=ZQHAAkZFNVDB81W9-0

Notes:
- responsive
- page URLs and user name are hardcoded, to be changed later

### CODE REVIEW

**Comment by choir241** [comment]
Screenshot (8)

On mobile view, the user dashboard doesn't match the figma design. Please fix this!

File: resources/js/Layouts/AuthenticatedLayout.jsx [hunk 1]
```diff
@@ -112,7 +120,11 @@ export default function AuthenticatedLayout({ header, children }) {
                     <div className="border-t border-gray-200 pb-1 pt-4 dark:border-gray-600">
                         <div className="mt-3 space-y-1">
                             <ResponsiveNavLink href="/">Home</ResponsiveNavLink>
-                            <ResponsiveNavLink href="" method="post" as="button">
+                            <ResponsiveNavLink
+                                href=""
+                                method="post"
```
Comment (choir241) [inline resources/js/Layouts/AuthenticatedLayout.jsx:125]: remove `method = "post"`

File: resources/js/Pages/Dashboard.jsx [hunk 2]
```diff
@@ -1,30 +1,67 @@
...
-                    <div className="overflow-hidden bg-white shadow-sm sm:rounded-lg dark:bg-gray-800">
-                        <div className="p-6 text-gray-900 dark:text-gray-100">
-                            You're logged in!
-                        </div>
-                    </div>
-                </div>
-            </div>
+            <h1 className="sr-only">Dashboard</h1>
+            <h2 className="leading-tight text-black font-bold text-5xl tracking-[-.005em] font-baskerville">
+                Hello, RICHARD
```
Comment (choir241) [inline resources/js/Pages/Dashboard.jsx:37]: Update so it uses the user name dynamically instead of static value

**Comment by emestabillo** [comment]
> Screenshot (8)
> 
> On mobile view, the user dashboard doesn't match the figma design. Please fix this!

Question: The screen width for the 'mobile' design on Figma is not at 375px, more like tablet width. Proceed with 2 items on mobile? @choir241

**Review by choir241 — CHANGES_REQUESTED** [review]

File: resources/js/Layouts/AuthenticatedLayout.jsx (hunk 1 above)
Comment (emestabillo) [reply resources/js/Layouts/AuthenticatedLayout.jsx]: Attribute removed

File: resources/js/Pages/Dashboard.jsx (hunk 2 above)
Comment (emestabillo) [reply resources/js/Pages/Dashboard.jsx]: Resolved. Used code posted by @choir241 on Discord chat

**Comment by choir241** [comment]
> Question: The screen width for the 'mobile' design on Figma is not at 375px, more like tablet width. Proceed with 2 items per row on mobile? @choir241

@emestabillo use both for mobile  --> tablet which is tailwind's sm --> md screen widths

**Comment by emestabillo** [comment]
> > Question: The screen width for the 'mobile' design on Figma is not at 375px, more like tablet width. Proceed with 2 items per row on mobile? @choir241
> 
> @emestabillo use both for mobile --> tablet which is tailwind's sm --> md screen widths

Addressed on the [most recent commit](https://github.com/choir241/Elecretanta/pull/39/commits/53a26941fc64a89613472c2b414308801677b2c1) @choir241

**Review by choir241 — APPROVED** [review]

**Pull request merged**